cloudconfig/userdatacfg_unix.go

```diff
@@ -122,6 +126,20 @@ func (w *unixConfigure) ConfigureBasic() error {
 			`sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers`,
 		)
 		w.addCleanShutdownJob(service.InitSystemSystemd)
+	case os.OpenSUSE:
+		w.conf.AddScripts(
+			// Mask and stop firewalld, if enabled, so it cannot start. See
+			// http://pad.lv/1492066. firewalld might be missing, in which case
+			// is-enabled and is-active prints an error, which is why the output
+			// is surpressed.
+			"systemctl is-enabled firewalld &> /dev/null && systemctl mask firewalld || true",
+			"systemctl is-active firewalld &> /dev/null && systemctl stop firewalld || true",
+			`sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers`,
+			//Scripts assume ubuntu group for ubuntu user...
+			`(grep ubuntu /etc/group) || groupadd ubuntu`,
```

Comment:
By default new users are created using "users" group. "ubuntu" group does not exists and adding it to OpenSUSEGroups has no effect (group is not created...). For this reason I added these two lines here.